### DISCUSSION TITLE
ImageGlass 8.2.5.16 - June

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1061] [General]
<img width="640" alt="ImageGlass 8.2.5.16" src="https://imageglass.org/upload/photo/release/8.2_1.jpg">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/8.2.5.16/total?color=%23932043&label=downloads&style=for-the-badge)](https://imageglass.org/download)

### 🌱 New:
- **ImageGlass official Discord server:** http://discord.io/imageglass 🍹
- Editing app: (#945)
  + Added support for `*.lnk` extension, environment variables (relative paths). You can use command for App path, e.g. `cmd`, `cmd.exe`, `imageglass`, ...
  + Added support for selecting multiple extensions to edit.
- Added support for auto-selecting the largest frame of ICO files (#994).

### 🔮 Improvements:
- Updated Magick.NET to 7.24.0.0
- Added color management for WebP and AVIF images (#1022), kudo to @datasone.
- Added support for printing all pages of TIFF file (#1031).
- Set minimum size of window in Window Fit mode (#814).

### 🐛 Bugfixes:
- Fixed an issue that cannot convert from WEBP to PNG (#1021).
- Fixed an issue where "Reached first image" state stops animated GIF from playing (#1020), thanks to @fire-eggs.
- Fixed an issue that opening an image pops up the message "Reached the first image" and only shows a blank screen until you go prev/next (#1019), thanks to @fire-eggs.
- Fixed an issue when rotating files and saving them causes corruption (#1033).
- Fixed an issue when `%AppData%\ImageGlass` does not exist, ImageGlass is unable to save user settings (#689).
- Fixed an issue that `Save image as...` crashed when file extension is empty (#1050).

### 🥒 Checksums:

File | SHA-1
-- | --
ImageGlass_8.2.5.16_x64.msi | `80e2f715b07ca3f052fa7f00c33b685742935958`
ImageGlass_8.2.5.16_x64.zip | `e5054b8dd5235325241c469936a30c5ebb12b90f`
ImageGlass_8.2.5.16_x86.msi | `c5d95aaaf99e569541fe732f3ae6902e1879266b`
ImageGlass_8.2.5.16_x86.zip | `ae76f7d91391954b53bf09af313d007489e62b51`

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/8.2.5.16'>ImageGlass 8.2.5.16 - June</a>.</em>

**Comment by xmha97** [comment]
Thanks for this release. Please add support for the *.url extension. (https://github.com/d2phap/ImageGlass/issues/787)

**Reply by d2phap** [reply]
I will look into it

**Comment by d2phap** [comment]
A hotfix v8.2.6.6 released! 
https://github.com/d2phap/ImageGlass/releases/tag/8.2.6.6
